broker-util/oo-admin-chk

```diff
@@ -239,7 +242,7 @@ if $total_errors == 0
   errcode = 0
 else
   print_message "FAILED", true
-  puts "Please refer to the oo-admin-repair tool to resolve some of these inconsistencies."
+  puts "If there are no suggestions printed with the error(s), please check the oo-admin-repair tool man page to see if it can resolve any inconsistencies."
```

Comment:
How about...

"Please refer to the oo-admin-repair tool man page to resolve some of these inconsistencies if no suggestion was provided with any error message(s)."

Do we want to point the users towards where the suggestions will be?  If there are 50+ errors, they may have missed the suggestions.